megalut/sim/stampgrid.py

```diff
@@ -214,9 +217,9 @@ def drawimg(catalog, simgalimgfilepath="test.fits", simtrugalimgfilepath=None, s
 		# We draw the desired profile
 		profile_type = params.profile_types[row["tru_type"]]
 		if profile_type == "Sersic":
-			gal = galsim.Sersic(n=float(row["tru_sersicn"]), half_light_radius=float(row["tru_rad"]), flux=float(row["tru_flux"]))
+			gal = galsim.Sersic(n=float(row["tru_sersicn"]), half_light_radius=float(row["tru_rad"]), flux=float(row["tru_flux"]), gsparams=gsparams)
```

Comment:
No prob. I also thought that this was global, but I stumbled on this:
http://stackoverflow.com/questions/24966419/fft-runtime-error-in-running-galsim
... and went for the painless quick fix. There is nothing dangerous at all here, in the worst case it will just be very slow.